browse/util/id_patterns.py

```diff
@@ -0,0 +1,329 @@
+r"""Patterns and functions to detect arXiv ids and Urls in text.
+
```

Comment:
Why is this coming back?

Reply:
I think this came back with merge from master; will remove